src/utils/geometries.py

```diff
@@ -75,4 +76,11 @@ def survey_polygon(row):
         diff = abs(comp - truth)
         print(f"{label}: computed={comp}, truth={truth}, diff={diff}") 
 
-    # This is also about 3 cm off. 
\ No newline at end of file
+    # This is also about 3 cm off. 
+
+def get_bounding_box(lat, lon, half_side_km=0.5):
```

Comment:
should we name this something that makes it more obvious how it's different from bounding_box? Maybe get_ee_bounding_box or something?